lib/bolt/config.rb

```diff
@@ -164,6 +164,8 @@ def update_from_file(data)
       @plugins = data['plugins'] if data.key?('plugins')
       @plugin_hooks.merge!(data['plugin_hooks']) if data.key?('plugin_hooks')
 
+      @future = data['future'] == true
```

Comment:
Is this a rubocopism?

Reply:
No, my thought was that `future` should only have two possible values:  `true` and `false. The only way it is `true` is if we set `future: true` in `bolt.yaml`. Should we instead *only* set the `$future` global when `data['future']` is true?

Reply:
Should we *only* set $future if `future: true`? Instead of always setting `$future` to either `true` or `false`?